src/pages/Bookmarked.css

```diff
@@ -69,19 +62,45 @@
     padding-left: 0.25em;
     opacity: 1;
 }
+.delete-default {
+    transition: all 1s ease 0.2s;
+    /*transition: opacity 0.2s ease;*/
+    transition-property: width, padding-left, padding-right, opacity;
+    box-sizing: border-box;
+    overflow: hidden;
+    padding-left: 0;
+    padding-right: 0;
+    width: 0px;
+    opacity: 0;
+    height: 0.85em;
+}
+.delete-revealed {
+    width: 1em;
+    padding-right: 1em;
+    padding-left: 0.25em;
+    opacity: 1;
+}
 .button-holder {
-    position: fixed;
+    /*position: fixed;
     padding-bottom: 1.25em;
     bottom: 0;
-    width: 100%;
+    width: 100%;*/
     display: flex;
     justify-content: center;
+    width: 75%;
 }
 .set-notification-button {
     --box-shadow: 0 0 5px 5px #fff;
 }
-.folder-list {
-
+.visible {
+    visibility: visible;
+    opacity: 1;
+    transition: opacity 0.5s ease-in;
+}
+.hidden {
+    visibility: hidden;
+    opacity: 0;
+    transition: visibility 0s 0.5s, opacity 0.5s ease-out;
 }
 .folders-menu-content {
     height: 100%;
```

Comment:
Git seems to hide Bookmarked.tsx diff by default, if you are doing a CR, please click load diff on Bookmarked.tsx